addToCollection.php

```diff
@@ -0,0 +1,104 @@
+<?php
+require_once 'functions.php';
+
+$db = getDB();
+$cleansedArray = cleanseData($_POST);
+
+if(!empty($cleansedArray)) {
```

Comment:
prob best to do an isset on the $_POST data keys (maybe even a function that returns true/false if all the keys are present) and then place your getDB() and cleanseData inside the if statement. This way you wont call these functions every time the page loads. only when you need them